api4/user.go

```diff
@@ -615,6 +603,8 @@ func autocompleteUsers(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	var autocomplete *model.UserAutocomplete
```

Comment:
I missed this in code review, and our unit tests aren't failing on the panic right now, but `autocomplete` is left `nil` here, unlike the previous allocation of `new(model.UserAutocomplete)`. I'm fixing this in `master` by just declaring it as a non-pointer.

Reply:
Thanks for fixing this. I relied to much on the CI. Should have checked the logs.